src/gpu/intel/jit/dsl/dsl.cpp

```diff
@@ -52,7 +52,10 @@ struct ctx_t {
                         local_id_type(), ir_builder_t::local_id(i));
                 group_ids_[i] = var_t::make(
                         group_id_type(), ir_builder_t::group_id(i));
+                subgroup_ids_[i] = def("sg_id",
```

Comment:
We are missing the dimension associated with the variable: 
```
"sg_id" + "012"[i]
```
Also, I don't think there is much benefit to shortening `subgroup -> sg`, that just seem like it obfuscates how the IR maps to the DSL interface.